iidrees/Events-Manager
# #157977156 Add Component Test

#### What does this PR do?
It ensures that the components are tested
#### Description of Task to be completed?
Complete component testing
#### How should this be manually tested?
Check the travis build information below 
Click on it to see the build information,
On the build log page, click on `restart build` button in order to see the test run
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
